engine/runtime/src/test/scala/org/enso/interpreter/test/semantic/NamedArgumentsTest.scala

```diff
@@ -257,16 +257,5 @@ class NamedArgumentsTest extends InterpreterTest {
 
       eval(code) shouldEqual 10
     }
-
-    "be assignable from Vectors" in {
-      val code =
-        """from Builtins import all
-          |
-          |main =
-          |    lam = (x=[1,3]) -> y -> y + Polyglot.get_array_element x 0 + Polyglot.get_array_element x 1
-          |    lam y=10
-          |""".stripMargin
-      eval(code) shouldEqual 14
-    }
```

Comment:
Is this test (and the other removed tests) obsolete or is it just removed, because it cannot be tested without the stdlib that contains vectors?

(If it is the latter, maybe it's worth marking these tests as ignored or gathering them somewhere so they could be re-introduced once #1046 is done?)

Reply:
A bit of both. With stdlib tests in place these are slowly becoming obsolete, as we are testing more and more involved programs.